### PR TITLE
add admonition to reflect changes

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
~~- [ ] A technical writer reviews the content or PR~~ xs size pr
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [x] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

Preview:
Snag_1533518

### CODE REVIEW

File: docs/semgrep-cloud-platform/user-management.md [hunk 1]
```diff
@@ -89,7 +89,13 @@ You have successfully joined an org in Semgrep Cloud Platform.
 
 ## Controlling access through roles 
 
-Access control in Semgrep Cloud Platform determines the resources and features that are available to users based on their role. This **role-based access control (RBAC)** feature is available for organizations on **Team/Enterprise tiers**.
+Access control in Semgrep Cloud Platform determines the resources and features that are available to users based on their role. This **role-based access control (RBAC)** feature is available for organizations on **Team or Enterprise tiers**.
+
+:::caution Changes to RBAC default roles
+Previously, new users in organizations with RBAC enabled were assigned a `member` role. Starting **June 14, 2023**, the default role is set through the **Defaults** configuration and defaults to `admin`.
```
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/user-management.md:95]: Something like "defaults to `admin` for any organization that did not previously make use of RBAC" is probably where we'll land here

Comment (leifdreizler) [reply docs/semgrep-cloud-platform/user-management.md:95]: I somewhat question if we need to make a change to the docs. 

- Customers that never used RBAC aren't impacted
- New customers aren't impacted
- Previous RBAC customers with 0 members in their deployment probably don't care
- Customers that did have members have had their default role set to member for their current deployment
- The few customers that had their behavior changed & invited someone during the 1 day window have been individually notified

I think the above message would really just apply to customers that are creating a second+ deployment that are used to the existing RBAC behavior on their previous deployment.

We already mention in the docs that the default behavior is `admin`, although that is lower in the doc.

Option for a simpler change:
[On line 108](https://github.com/returntocorp/semgrep-docs/pull/1076/files#diff-2c21bbf6c1c2f90aa4e383bdf649ed1239454bc0896df6abb7cca9c1ca8a17e7R108), add "New organizations are created with a default role of `admin`".

**Review by leifdreizler — APPROVED** [review]

File: docs/semgrep-cloud-platform/user-management.md [hunk 2]
```diff
@@ -89,7 +89,13 @@ You have successfully joined an org in Semgrep Cloud Platform.
 
 ## Controlling access through roles 
 
-Access control in Semgrep Cloud Platform determines the resources and features that are available to users based on their role. This **role-based access control (RBAC)** feature is available for organizations on **Team/Enterprise tiers**.
+Access control in Semgrep Cloud Platform determines the resources and features that are available to users based on their role. This **role-based access control (RBAC)** feature is available for organizations on **Team or Enterprise tiers**.
```
Comment (leifdreizler) [inline docs/semgrep-cloud-platform/user-management.md:92]:
```suggestion
Access control in Semgrep Cloud Platform determines the resources and features that are available to users based on their role.
```

Comment (leifdreizler) [reply docs/semgrep-cloud-platform/user-management.md:92]: This is available to all customers 🙂

File: docs/semgrep-cloud-platform/user-management.md (hunk 1 above)
Comment (s-santillan) [reply docs/semgrep-cloud-platform/user-management.md:95]: @leifdreizler suggestion: how about keeping this admonition for a month? 

For @armchairlinguist we still default to admin for any org regardless of case (new, old, previously didn't use RBAC) so I think the original phrasing is fine

Comment (armchairlinguist) [reply docs/semgrep-cloud-platform/user-management.md:95]: Hmm, I don't know if we have different definitions of "default". We've set a default of "Member" for folks previously making use of RBAC. The "default default" is still admin.

I think Leif's change might be a good choice.

Comment (s-santillan) [reply docs/semgrep-cloud-platform/user-management.md:95]: I'll add Leif's suggestion, remove the admonition a month, after which we can mention the change in behavior in the release notes. [Reason being that we once deprecated an old Jira integration](https://semgrepinc.slack.com/archives/C0370TA46PL/p1678301197065599), and people were looking for when that happened.

LMK what yall think.

Comment (leifdreizler) [reply docs/semgrep-cloud-platform/user-management.md]: sgtm!

**Comment by s-santillan** [comment]
Thanks! Merging.

**Pull request merged**